python_modules/dagster/dagster_tests/core_tests/resource_tests/verifiable_resources/test_verifiable_resource.py

```diff
@@ -0,0 +1,119 @@
+from typing import cast
+
+from dagster import (
+    ConfigurableResource,
+    ConfigVerifiable,
+    Definitions,
+    EnvVar,
+    VerificationResult,
+    VerificationStatus,
+)
+from dagster._config.structured_config.resource_verification import (
+    resource_verification_job_name,
+    resource_verification_op_name,
+)
+from dagster._core.test_utils import environ
+
+
+def test_config_verifiable_job_basic() -> None:
+    class MyVerifiableResource(ConfigurableResource, ConfigVerifiable):
+        a_str: str
```

Comment:
nit: variables that describe function in test a bit better such as `failure_switch`